dashboard/app/controllers/rubrics_controller.rb

```diff
@@ -35,15 +35,31 @@ def update
     @lesson = @rubric.lesson
 
     if @rubric.update(rubric_params)
-      redirect_to edit_rubric_path(@rubric.id), notice: 'Rubric was successfully updated.'
+      render json: @rubric
+      # KT NOTE FOR FUTURE WORK: tried redirect, couldn't get it to work.  Need to add notice. redirect_to edit_rubric_path(@rubric.id), notice: 'Rubric was successfully updated.'
```

Comment:
I'd love feedback about the best way to take notes of this type of thing.   Notes like this help me track my own thinking but I know gum up the files if others work on them.  Given that this is my next task, I am wondering if this is an OK way/place to note this sort of thing.

Reply:
I usually use a TODO, ideally with the jira task key if possible.

So in this case, maybe `TODO(KT) [AITT-59]: add notice that rubric was successfully updated`

I know that loses some of the notes, though. If you're definitely going to fix this really soon, this is okay, but I wouldn't want notes like this to live in our codebase for too long.

Reply:
I do like this solution, especially for this since it doesn't seem like NEED TO HAVE work.